### PR TITLE
fix #80705 sematext.com

### LINKED ISSUE #80705: sematext.com
### Issue URL (Incorrect Blocking)
 [https://sematext.com/blog/journald-logging-tutorial/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsematext.com%2Fblog%2Fjournald-logging-tutorial%2F)
### Comment
Username: @chirag127
> broken header

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#80705

Also grouped some other domain to common `#wpfront-notification-bar-spacer` rule.

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1174,6 +1174,7 @@ lightnovelpub.com,lightnovelworld.com#$#.chapter-content { -webkit-touch-callout
 mediafire.com#$#.nonDLContentWrap { min-height: 1300px !important; }
 lanacion.com.ar#$#.sidebar__aside { background: none!important; }
 goha.ru#$##site { margin-top: 150px !important; }
+sematext.com#$#.navbar-fixed-top { top: 0 !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt:1177]: why not to popups.txt?

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/tweaks.txt:1177]: I've only moved navbar so the gap won't show

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/tweaks.txt:1177]: @Alex-302

Comment (Alex-302) [reply AnnoyancesFilter/sections/tweaks.txt:1177]: it is related to `#wpfront-notification-bar-spacer`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**